src/i18n/locales/he.js

```diff
@@ -0,0 +1,241 @@
+const hebrew = {
+    name: "עברית",
```

Comment:
Could you please rename the name field to 'Hebrew'

Reply:
Thanks for notice, fixed.